deploy.py

```diff
@@ -334,7 +339,7 @@ def main():
         # script is running on CI, proceed with auth and helm setup
         if args.cluster == 'ovh':
             setup_auth_ovh(args.release, args.cluster)
-        elif args.cluster == 'turing':
+        elif args.cluster in list(AZURE_RGs.keys()):
```

Comment:
There's a simpler check for whether something is in the keys of a dict:

```suggestion
        elif args.cluster in AZURE_RGs:
```